sidh/sidhp751/sidhp751.go

```diff
@@ -59,68 +94,44 @@ var bobIsogenyStrategy = [maxBob]int{0, 1, 1, 2, 2, 2, 3, 3, 4, 4, 4, 5, 5, 5, 6
 	88, 88, 86, 86, 86, 93, 90, 90, 92, 92, 92, 93, 93, 93, 93, 93, 97, 97, 97, 97,
 	97, 97}
 
-// Bob's public key.
-type SIDHPublicKeyBob struct {
-	affine_xP   ExtensionFieldElement
-	affine_xQ   ExtensionFieldElement
-	affine_xQmP ExtensionFieldElement
-}
-
 // Read a public key from a byte slice.  The input must be at least 564 bytes long.
 func (pubKey *SIDHPublicKeyBob) FromBytes(input []byte) {
```

Comment:
This looks like it could be a generic function in the sidh package.

Reply:
It is one of the things that I wasn't sure about. Since in any other DH key-exchange package this kind of functionality is not available, I thought it would make more sense to leave it in the lower implementation. But I will add it to the sidh package since we need to use it inside sike implementation for instance.